relay-event-normalization/src/event.rs

```diff
@@ -773,10 +773,18 @@ fn normalize_breakdowns(event: &mut Event, breakdowns_config: Option<&Breakdowns
 /// Normalizes incoming contexts for the downstream metric extraction.
 fn normalize_contexts(contexts: &mut Annotated<Contexts>) {
     let _ = processor::apply(contexts, |contexts, _meta| {
+        // Reprocessing context sent from SDKs must not be accepted, it is a Sentry-internal
+        // construct.
+        // This does not run on renormalization anyway.
```

Comment:
```suggestion
        //[`normalize`] does not run on renormalization anyway.
```

Could we rename `normalize` to something like `primary_normalization` to make it more clear from the name that it's the part of normalization that runs when `! is_renormalize`? Might also be worth adding that piece of information to the function's doc comment.

Reply:
I agree it can be a bit confusing with all the normalization steps moving around. I think it should be clearer and easier to follow once there's a single normalization step. I'd like to avoid such names as it implies there are other normalization steps, or assume that's the only piece running on `!is_renormalize`. Happy to revisit this decision after migrating the store normalizer.

Addressed the comment suggestion in [389b5e0](https://github.com/getsentry/relay/pull/3054/commits/389b5e0c10ee504aa2e078b6c2ae42933beb1c04).